### PR TITLE
Rs 2022 01 23

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3C.txt [hunk 1]
```diff
@@ -106,32 +118,35 @@ I'm begging you, please try to be a bit more discreet! You stand out too much ju
...
 -- 「―――なによ、勝手なことばっかり言って。
-"―――What's up with you, running your mouth like that.
+"―――What's up with you, mouthing off to me like that.
 }
 [sha:2c9ad6f8e9302576913911df06dba4a91d4beb9a]{
 -- Page 217, Offset 17103.
 -- だいたい居るだけで目立つってどうしてよ。わたし、外見は人間と変わらないでしょう？」
-Besides, what do you mean by me standing out? It's not like I can change my appearance into a human being, right?"
+Besides, what do you mean I stand out just by existing?
+ It's not like I can change my appearance into that of a human being, right?"
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3C.txt:142]: I'm pretty sure  外見は人間と変わらないでしょう means the opposite of how it's translated right now. "My outward appearance is that of a human being, right?"

File: script/Arcueid/Day 7/07_00_ARC07_3C.txt [hunk 2]
```diff
@@ -151,7 +167,10 @@ Whether it's because my reply left no room for discourse, it seems that Arcueid
 [sha:b9ff6fce2c8e3ce43b0cbb68b527cc34c3745e57]{
 -- Page 219, Offset 17108.
 -- こういう修羅場ははじめてだったから、気が動転してたんだ」
-I've never experienced such a bloodbath before, so I got a bit flustered."
+// 修羅場 => location of the fight between the Devas and Asuras,
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3C.txt:170]: I honestly had never heard the literal meaning before, but 修羅場 is a pretty normal way to say "scene of carnage", it's somewhat common describing like messy love triangles and stuff

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 3]
```diff
@@ -241,12 +243,13 @@ I don't think it's something to be that upset about."
...
 -- 「―――なによ、勝手なことばっかり言って。
-"―――What's up with you, running your mouth like that.
+"―――What's up with you, mouthing off to me like that.
 }
 [sha:2c9ad6f8e9302576913911df06dba4a91d4beb9a]{
 -- Page 453, Offset 15885.
 -- だいたい居るだけで目立つってどうしてよ。わたし、外見は人間と変わらないでしょう？」
-Besides, what do you mean by me standing out? It's not like I can change my appearance into a human being, right?"
+Besides, what do you mean I stand out just by existing?
+ It's not like I can change my appearance into that of a human being, right?"
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:252]: same as above

**Review by Avuxo — APPROVED** [review]
one lingering mis-TL but otherwise good

File: script/Arcueid/Day 7/07_00_ARC07_3C.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3C.txt:170]: Ah, interesting. Bloodbath / active carnage just felt off since no blood was spilled (barely)

**Pull request merged**